### PR TITLE
Mobile Setup-Head-body Version

### PR DESCRIPTION
**Pull request by ShaddyMore**
## Mobile Setup Middle Version

In this project, I did the following:
- Created the header part of the mobile setup
- Created the works section
- Created an about me section
- styled HTML file to look good

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+  <title>Middle-Version-Mobile Setup</title>
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;800&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <header class="head"> 
+      <nav class="nav-bar"> 
+          <a class="logo" href="#"><h3>My Logo<h3></a>
+          <img class="union" src="/images/Union.png" alt="Hamburger menu">
```
Comment (OyePriscilla) [inline index.html]: - [ ] Please kindly take note that all your images are not showing; you missed the dot (.) notation before the /.
Should be `Hamburger menu` not `Hamburger menu`. 😄 Please implement this for all your project images.

## Your Design
127 0 0 1_5500_Mobile-Setup-Head-Middle_index html(mobile)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+              </ul>
+            </li>
+            <li class="my-skill">
+              <div class="skill-container">
+                <h3 class="skill-title">Frameworks</h3>
+                <button class="btn-chev" type="button">
+                    <img src="./images/Chevron-right.png" alt="Dropdown-Right">
+                </button>
+              </div>
+              <div class="line-1"><hr></div>
```
Comment (OyePriscilla) [inline index.html]: - [ ] Kindly adjust this according to the design template.

## Your Design 

b6


## Figma Design 

crr7

File: inde.html [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+         <li><a class="icon icon-linkedin" href="Linkedin"><img src="/icon-social/Linkedin.png" alt="Linkedin icon"></a></li>
+         <li><a class="icon icon-medium" href="Messenger"><img src="/icon-social/Messenger.png" alt="Messenger icon"/></a></li>
+         <li><a class="icon icon-github" href="Github"><img src="/icon-social/Github.png" alt="Github icon"/><a></li>
+         <li><a class="icon icon-iOS" href="iOS"><img src="/icon-social/iOS.png" alt="iOS icon"></a></li>
+      </ul>
+    </section> 
+    <section class="work-container">
+        <div class="work-items">
+            <a class="design" href="#"><img src="/images/Designer.png" alt="Designer"></a>
+            <h1 class="work-title">Tonic</h1>
```
Comment (OyePriscilla) [inline inde.html]: - [ ] Kindly change this header tag to any other header tag (h2,h3 ...); it's part of best practices to have just one h1 tag in an HTML document purposely for SEO optimization. Please do this for all your cards and `About me `section as well. 😄

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @ShaddyMore    😃 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Descriptive commit messages 😄 
LInters checks passed ✔️ 
GitHub flow observed ✔️ 

## Required Changes ♻️

- [ ] @ShaddyMore I noticed you have two .html file with the same contents; kindly remove one to make your project cleaner.

Kindly check the inline comments below for the required changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+                <li class="lang"><a href="#">javascript</a></li>
+             </ul>
+             <div>
+                  <button type="submit" class="project-btn">See Project</button>
+             </div>
+        </div>
+        <div class="work-items">
+              <a class="design" href="#"> <img src="./images/art-printing.png" alt="Art printing"></a>
+                  
+              <h1 class="work-title">Multi-Post
```
Comment (bienvenuushindi) [inline index.html]: :red_circle:  Please make sure you change this header tag to any other header tag (h2,h3 ...); As requested by the previous reviewer your HTML should not have more than one  `<h1> ` tag.  H1 on the page indicates what the entire web page is about. And the rest of the headings follow a hierarchical structure, with H2 for each topic section and any subsections to those being indicated with H3 and if the subtopic has subtopics then going down as low as H4.

I think this code should be `<h3 class="work-title">Multi-Post <br>Stories </h3>`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+                <li class="lang">css</a></li>
+                <li class="lang">javascript</a></li>
+             </ul>
+             <div>
+             <button type="submit" class="project-btn">See Project</button>
+             </div>
+        </div>
+        <div class="work-items">
+             <a class="design" href="#"> <img src="./images/gymfit.png" alt="Gymfit"></a>
+             <h1 class="work-title">Tonic </h1>
```
Comment (bienvenuushindi) [inline index.html]: :red_circle:  Please make sure you change this header tag to any other header tag (h2,h3 ...); As requested by the previous reviewer your HTML should not have more than one  `<h1> ` tag.  H1 on the page indicates what the entire web page is about. And the rest of the headings follow a hierarchical structure, with H2 for each topic section and any subsections to those being indicated with H3 and if the subtopic has subtopics then going down as low as H4.

I think this code should be `<h3 class="work-title">Tonic </h3>`

File: index.html [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+               <li class="lang"><a href="#">css</a></li>
+               <li class="lang"><a href="#">javascript</a></li>
+            </ul>
+            <div>
+                <button type="submit" class="project-btn">See Project</button>
+            </div>
+        </div>           
+        <div class="work-items">
+           <a class="design" href="#"> <img src="./images/Avialability.png" alt="Availability"></a>
+             <h1 class="work-title">Multi-Post
```
Comment (bienvenuushindi) [inline index.html]: :red_circle:  Please make sure you change this header tag to any other header tag (h2,h3 ...); As requested by the previous reviewer your HTML should not have more than one  `<h1> ` tag.  H1 on the page indicates what the entire web page is about. And the rest of the headings follow a hierarchical structure, with H2 for each topic section and any subsections to those being indicated with H3 and if the subtopic has subtopics then going down as low as H4.

I think this code should be `<h3 class="work-title">Multi-Post <br>Stories </h3>`

File: index.html [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+         <li><a class="icon icon-linkedin" href="Linkedin"><img src="./icon-social/Linkedin.png" alt="Linkedin icon"></a></li>
+         <li><a class="icon icon-medium" href="Messenger"><img src="./icon-social/Messenger.png" alt="Messenger icon"/></a></li>
+         <li><a class="icon icon-github" href="Github"><img src="./icon-social/Github.png" alt="Github icon"/><a></li>
+         <li><a class="icon icon-iOS" href="iOS"><img src="./icon-social/iOS.png" alt="iOS icon"></a></li>
+      </ul>
+    </section> 
+    <section class="work-container">
+        <div class="work-items">
+            <a class="design" href="#"><img src="./images/Designer.png" alt="Designer"></a>
+            <h1 class="work-title-1">Tonic</h1>
```
Comment (bienvenuushindi) [inline index.html]: :red_circle:  Please make sure you change this header tag to any other header tag (h2,h3 ...); As requested by the previous reviewer your HTML should not have more than one  `<h1> ` tag.  H1 on the page indicates what the entire web page is about. And the rest of the headings follow a hierarchical structure, with H2 for each topic section and any subsections to those being indicated with H3 and if the subtopic has subtopics then going down as low as H4.

I think this code should be `<h3 class="work-title-1">Tonic</h3>`

File: index.html [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+            </li>
+            <li class="my-skill">
+              <div class="skill-container">
+                <div class="skill-title">Skills</div>
+                <button class="btn-chev" type="button">
+                    <img src="./images/Chevron-right.png" alt="Dropdown-Right">
+                </button>
+              </div>
+              <div class="line-1"><hr></div>
+            </li>
```
Comment (bienvenuushindi) [inline index.html:183]: :red_circle: You have done well adjusting the languages block according to the design template :+1: . Please make sure you align correctly the Frameworks and Skills items to match the Figma Design

Your Design
still

Figma Design

fima

File: index.html [hunk 9]
```diff
@@ -0,0 +1,186 @@
...
+
+            <a href="#" class="let-connect">LET'S CONNECT </a>
+
+            <ul class="icons-social-1">
+                <li><a class="icon icon-Github" href="#"> <img src="../icon-social/Github.png" alt="Github"/><a></a></li>
+                <li><a class="icon icon-Linkedin" href="#"> <img src="../icon-social/Linkedin.png" alt="Linkedin"></a></li>
+                <li><a class="icon icon-iOS" href="#"> <img src="../icon-social/iOS.png" alt="iOS"></a> </li>
+                <li><a class="icon icon-Twitter" href="#"><img src="../icon-social/Twitter.png" alt="Twitter"></a></li>
+                <li><a class="icon icon-Messenger" href="#"> <img src="../icon-social/Messenger.png" alt="Messenger"/></a></li>
+           </ul>
```
Comment (bienvenuushindi) [inline index.html]: :red_circle:  As you can below none of these icons is displaying. You should keep one dot `.` before / to index the current directory. Kindly use one dot like this  `Github<a>` and not 
`Github<a>`. Please update all the images path under this  section :hugs: 

Your Design
iconnotdispl

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
### Hi @ShaddyMore,

Good job implementing some of the required changes!
There are some issues that you still need to work on to go to the next project but you are almost there!



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- [ ] N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Olamarx — APPROVED** [review]
Hi @ShaddyMore, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Good commit messages 👍🏼 
- All linters passed ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**